target/linux/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -36,6 +36,13 @@ platform_do_upgrade() {
 	redmi,ax6|\
 	xiaomi,ax3600|\
 	xiaomi,ax9000)
+	
+		# Circunvent a random issue with WPAD taking a huge time to stop while upgrading with ath11 driver, leading the sysupgrade role to failure
+		/etc/init.d/wpad stop 2>/dev/null
```

Comment:
service

Reply:
I assume is to force any zombie wpad thread to terminate, as a redundancy to the previous stop command.